public/javascripts/notebook/observable.js

```diff
@@ -113,31 +113,48 @@ return new function () {
 
 
 
-  this.makeObservableHelper = function (id, kind, initialValue) {
+  this.makeObservableHelper = function (id, kind, initialValue, df) {
+    var _df = df
+    if (_.isUndefined(_df)) {
+      _df = function(publishValue) {
+        this.df.done(function(ob) {
+          ob(publishValue);
+        });
+      }
+      _df.df = $.Deferred();
+      _df.subscribe = function(subscriber) {
+        this.df.done(function(ob) {
+          ob.subscribe(subscriber);
+        });
+      };
```

Comment:
btw: `Uncaught TypeError: Observable.makeObservable(...).extend is not a function`